ansible-inventory-origin-15-aio

```diff
@@ -0,0 +1,24 @@
+
+[OSEv3:children]
+nodes
+masters
+
+[OSEv3:vars]
+ansible_connection=local
+openshift_release=v1.5
+openshift_hosted_logging_deploy=true
+openshift_logging_install_logging=true
+short_version=1.5
+openshift_deployment_type=origin
+openshift_logging_namespace=logging
+openshift_logging_image_prefix=docker.io/openshift/origin-
+openshift_master_identity_providers=[{'challenge': 'true', 'login': 'true', 'kind': 'AllowAllPasswordIdentityProvider', 'name': 'allow_all'}]
+openshift_logging_es_cluster_size=1
+openshift_logging_image_version=v1.5.0-rc.0
+deployment_type=origin
+
+[nodes]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
+
+[masters]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
```

Comment:
Why is this (and the host under `[nodes]`) localhost? We only support local deployment?